curvefs/src/mds/fs_manager.cpp

```diff
@@ -380,7 +380,18 @@ FSStatusCode FsManager::CreateFs(const ::curvefs::mds::CreateFsRequest* request,
         LOG(ERROR) << "CreateFs fail, insert root inode fail"
                    << ", fsName = " << fsName
                    << ", ret = " << FSStatusCode_Name(ret);
-        // TODO(wanghai): need delete partition if created
+        // delete partition if created
+        if (TopoStatusCode::TOPO_OK == topoRet) {
+            if (TopoStatusCode::TOPO_OK !=
```

Comment:
when `ret` is equal to `FSStatusCode::CREATE_PARTITION_ERROR`, no need to delete partition.

Reply:
> when `ret` is equal to `FSStatusCode::CREATE_PARTITION_ERROR`, no need to delete partition.

Fixed.